### DISCUSSION TITLE
Google Calendar notification option

### DISCUSSION THREAD

**Discussion by elimoler** [Vaccarini-Lorenzo/iCalObsidianSync#28] [Q&A] [answered]
Hello, love this great new plugin for Obsidian! I like being able to write naturally to sync events to my Google Calendar. Google Calendar has built-in notifications for events. Is there a way to add a notification for an event from Obsidian? It could be something simple like always set notification for 4 hours before any event entry.

**Comment by Vaccarini-Lorenzo** [comment, marked as answer]
Hi!
Thank you for opening this discussion. The feature is surely implementable, I just have to find the time (or some collaborators) to do so. I'll keep you posted!

**Reply by elimoler** [reply]
Gotcha! it's not exactly a necessity, so I understand the need to prioritize other things. Thanks.
